### PR TITLE
docs: use repo-metadata to generate the README

### PR DESCRIPTION
**Pull request by JustinBeckwith**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,54 +1,122 @@
+[//]: # "This README.md file is auto-generated, all changes to this file will be lost."
+[//]: # "To regenerate it, use `python -m synthtool`."
 <img src="https://avatars2.githubusercontent.com/u/2810941?v=3&s=96" alt="Google Cloud Platform logo" title="Google Cloud Platform" align="right" height="96" width="96"/>
 
-# google-p12-pem
+google-p12-pem
```
Comment (bcoe) [inline README.md]: whoops, it actually expect the title in `.readme-partials.json` to include the `#`.

File: README.md [hunk 2]
```diff
@@ -1,54 +1,122 @@
...
+
+Apache Version 2.0
+
+See [LICENSE](https://github.com/googleapis/google-p12-pem/blob/master/LICENSE)
+
+
+
+[shell_img]: https://gstatic.com/cloudssh/images/open-btn.png
+[projects]: https://console.cloud.google.com/project
+[billing]: https://support.google.com/cloud/answer/6293499#enable-billing
```
Comment (bcoe) [inline README.md]: looks like we did the right thing and don't include any billing links if `requires_billing` is straight up missing? good on us!

File: .readme-partials.yaml [hunk 3]
```diff
@@ -1,4 +1,4 @@
-title: google-p12-pem
+title: # google-p12-pem
```
Comment (bcoe) [inline .readme-partials.yaml]: I think that the parser screwed this up and didn't populate `title`, hence `node.js client` being appended.

```
title: |-
    # google-p12-pem
```

should have worked ... maybe it saw the # as a comment?

Comment (JustinBeckwith) [reply .readme-partials.yaml]: *meep*.  Looks like maybe that was the case.  What do you suggest here?  Mind making a change (when you get a chance) to have synthtool add the `#` in front of the generated title?

**Pull request merged**